mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -674,12 +674,13 @@ object LogisticRegressionModel extends MLReadable[LogisticRegressionModel] {
 
       val dataPath = new Path(path, "data").toString
       val data = sparkSession.read.format("parquet").load(dataPath)
-        .select("numClasses", "numFeatures", "intercept", "coefficients").head()
+        .select("numClasses", "numFeatures", "intercept", "coefficients")
+      val row = MLUtils.convertVectorColumnsToML(data, "coefficients").head()
```

Comment:
Btw, if we are going to update this PR, it is more readable to write

~~~scala
val Row(numClasses: Int, numFeatures: Int, intercept: Double, coefficients: Vector) = data.head()
~~~